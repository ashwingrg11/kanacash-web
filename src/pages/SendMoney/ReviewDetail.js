import React, { Component } from "react";
import { Container, Row, Col, Card } from "react-bootstrap";
import BackArrow from "../../assets/images/backArrow.svg";
import "./sendMoney.css";
import PropTypes from "prop-types";
import { withRouter } from "react-router-dom";
import SendingMoneyDetail from "../../containers/transaction/SendingMoneyDetail";
import { connect } from "react-redux";
import SenderUserDetail from "../../containers/transaction/SenderUserDetail";
import ReciverUserDetail from "../../containers/transaction/ReciverUserDetail";
import CreditCardSvg from "../../assets/images/creditcard.svg";
import BankLogoSvg from "../../assets/images/BankLogoSvg.svg";
import BankoutSvg from "../../assets/images/bank-out.svg";
import SenderBankInfo from "../../containers/transaction/SenderBankInfo";
import SenderCardInfo from "../../containers/transaction/SenderCardInfo";
import ReciverBankInfo from "../../containers/transaction/ReciverBankInfo";
import { createTransaction } from "../../services/axios/transactionApi";
import AuthorizationModal from "../../containers/transaction/AuthorizationModal";
import SendMoneyHeader from "../../containers/transaction/SendMoneyHeader";
import {
  clearTransactionData,
  getTransactionAmount,
} from "../../store/actions/TrasactionAction";
import { getSenderTransactionLimit } from "../../store/actions/senderDetail";
import RightArrow from "../../assets/images/icons/RightArrow.svg";
import Footer from "../../containers/layout/Footer";
import Header from "../../containers/layout/Header";
import CustomAlert from "../../components/Alert/CustomAlert";
import CustomSpinner from "../../components/CustomSpinner";
import ROUTES from "../../assets/Routes/Routes";
import Disclaimer from "../../components/Disclaimer/Disclaimer";
class ReviewDetail extends Component {
  constructor(props) {
    super(props);
    this.state = {
      showAuthorizationModal: false,
      errorMessage: "",
      variant: "",
      showAlert: false,
      loading: false,
      transferFee: "",
      isSubmitting: false,
    };
  }
  componentWillUnmount = () => {
    this.setState({
      showAuthorizationModal: false,
      errorMessage: "",
      variant: "",
      showAlert: false,
      loading: false,
      transferFee: "",
    });
  };

  closeModal = () => {
    this.setState({ showAuthorizationModal: false });
  };

  closeAlert = () => {
    this.setState({ showAlert: false, variant: "", errorMessage: "" });
  };

  sendMoney = () => {
    this.setState({ showAuthorizationModal: true });
  };

  componentDidMount = () => {
    console.log("transaction", this.props.transactionDetail);
    let transferFee;
    let senderPaymentMethod;
    if (this.props.senderPaymentMethod.payoutMethod === "BANK") {
      senderPaymentMethod = "BANK";
    } else {
      senderPaymentMethod = "CARD";
    }
    // console.log("this.", this.props.transactionDetail.feeAmount);
    if (this.props.transactionDetail.feeAmount !== undefined) {
      const findTransferFee = this.props.transactionDetail.feeAmount.find(
        (item) => {
          return (
            item.senderPayment === senderPaymentMethod &&
            item.receiverPayout ===
              this.props.receiverPaymentMethod.payoutMethod
          );
        }
      );
      transferFee = findTransferFee;
      let body = {
        feeAmount: this.props.transactionDetail.feeAmount,
        exchangeRate: this.props.transactionDetail.exchangeRate,
        senderAmount: this.props.transactionDetail.senderAmount,
        recipientAmount: this.props.transactionDetail.recipientAmount,
        recipientCurrency: "USD",
        destinationCountry: this.props.transactionDetail.destinationCountry,
        minFee: findTransferFee.feeRange,
        selectedCountry: this.props.transactionDetail.selectedCountry,
      };
      this.props.getTransactionAmount(body);
      localStorage.setItem("transactionDetail", JSON.stringify(body));
    } else {
      transferFee = { feeRange: this.props.transactionDetail?.minFee };
    }

    this.setState({ transferFee });
  };

  continueSendMoney = () => {
    this.setState({ isSubmitting: true });
    let body = {};
    // body = this.props.transactionDetail;
    body.recipientId = this.props.receiverInfo.beneficiaryId;
    body.payoutMethod = this.props.receiverPaymentMethod.payoutMethod;
    body.fundingSource = this.props.senderPaymentMethod.payoutMethod;
    body.senderFundingAccountId = this.props.senderPaymentMethod.payoutDetail.id;
    body.recipientBankId = this.props.receiverPaymentMethod.payoutDetail.beneficiaryAccountId;
    body.exchangeRate = this.props.transactionDetail.exchangeRate;
    body.senderAmount = parseFloat(this.props.transactionDetail.senderAmount);
    body.recipientAmount = this.props.transactionDetail.recipientAmount;
    body.recipientCurrency = this.props.transactionDetail.selectedCountry.currency.code;
    body.destinationCountry = this.props.transactionDetail.destinationCountry;
    body.feeAmount = parseFloat(this.state.transferFee.feeRange);
    body.remittancePurpose = "Land Purchase";
    body.ipAddress = "27.34.104.241";

    createTransaction(body)
      .then((res) => {
        this.props.getSenderTransactionLimit();
        localStorage.removeItem("activeSideBar");
        localStorage.removeItem("transactionDetail");
        localStorage.removeItem("beneficiaryInfo");
        localStorage.removeItem("beneficiaryDetail");
        localStorage.removeItem("senderPaymentDetail");
        this.setState({ isSubmitting: false });
        const pushObject = {
          pathname: ROUTES.CONGRATULATION_PAGE,
          state: this.props.receiverInfo,
        };
        this.props.clearTransactionData();
        this.props.history.push(pushObject);
      })
      .catch((err) => {
        this.setState({ isSubmitting: false });
        // if (typeof err.response.data.message === "object") {
        //   this.setState({
        //     errorMessage: `${JSON.parse(err.response.data.message).message}`,
        //     showAlert: true,
        //     variant: "danger",
        //   });
        // } else {
        //   this.setState({
        //     errorMessage: `${err.response.data.message}`,
        //     showAlert: true,
        //     variant: "danger",
        //   });
        // }
      });
  };
  render() {
    return (
      <>
        <Header />
        <div className="shadow-line"></div>
        <section className="theme-inner">
          <Container>
            <SendMoneyHeader />
            <Row>
              <Col className="steps-content-block">
                <h2 className="sub-title text-md-center text-lg-left">
                  {" "}
                  Here is your transaction summary
                </h2>
                <p
                  style={{ fontSize: "18px" }}
                  className="text-md-center text-lg-left"
                >
                  Please review to make sure everything is correct before
                  proceeding.
                </p>
                <Row>
                  <Col
                    md={{ span: 8, offset: 2 }}
                    sm="12"
                    lg={{ span: 7, offset: 0 }}
                    className="pr-lg-0 mb-md-5 mb-lg-0"
                  >
                    <Row>
                      <Col className="pr-xl-0">
                        {this.props.receiverInfo && (
                          <Card style={{ padding: "20px" }}>
                            {this.state.loading && <CustomSpinner />}
                            <Row>
                              <Col
                                sm="5"
                                xs="5"
                                className="review-info sender-review"
                              >
                                <h3 className="review-title green-title">
                                  {" "}
                                  Sender Information
                                </h3>
                                <SenderUserDetail
                                  userDetail={this.props.user}
                                />
                              </Col>
                              <Col sm="2" xs="2" className="arrow text-center">
                                {" "}
                                <Row style={{ marginTop: "30px" }}>
                                  <Col>
                                    <img src={RightArrow} alt="RightArrow" />
                                  </Col>
                                </Row>
                              </Col>
                              <Col
                                sm="5"
                                xs="5"
                                className="review-info receiver-review"
                              >
                                <h3 className="review-title green-title">
                                  {" "}
                                  Recipient Information
                                </h3>
                                <ReciverUserDetail
                                  receiverInfo={this.props.receiverInfo}
                                />
                              </Col>
                            </Row>
                            <Row>
                              <Col
                                sm="5"
                                xs="5"
                                className="review-info sender-review"
                              >
                                <div className="review-sub-block">
                                  <h3 className="review-title">
                                    Payment Method{" "}
                                  </h3>
                                  <div className="review-method">
                                    {this.props.senderPaymentMethod
                                      .payoutMethod === "CARD" ? (
                                      <>
                                        <img
                                          alt="CreditCardSvg"
                                          src={CreditCardSvg}
                                        />
                                        Credit Card{" "}
                                        <span
                                          onClick={() =>
                                            this.props.history.push(
                                              `${ROUTES.PAYMENT_INFO}`
                                            )
                                          }
                                        >
                                          ( Change )
                                        </span>{" "}
                                      </>
                                    ) : (
                                      <>
                                        <img
                                          alt="BankLogoSvg"
                                          src={BankLogoSvg}
                                        />
                                        Bank{" "}
                                        <span
                                          onClick={() =>
                                            this.props.history.push(
                                              `${ROUTES.PAYMENT_INFO}`
                                            )
                                          }
                                        >
                                          ( Change )
                                        </span>{" "}
                                      </>
                                    )}{" "}
                                  </div>
                                </div>
                                <div className="review-sub-block payment-info-show">
                                  {this.props.senderPaymentMethod
                                    .payoutMethod === "CARD" ? (
                                    <SenderCardInfo
                                      cardInfo={
                                        this.props.senderPaymentMethod
                                          .payoutDetail
                                      }
                                      userDetail={this.props.user}
                                    />
                                  ) : (
                                    <SenderBankInfo
                                      bankInfo={
                                        this.props.senderPaymentMethod
                                          .payoutDetail
                                      }
                                      userDetail={this.props.user}
                                    />
                                  )}
                                </div>
                              </Col>
                              <Col sm="2" xs="2" className="arrow text-center">
                                {" "}
                                <Row style={{ marginTop: "60px" }}>
                                  <Col>
                                    <img src={RightArrow} alt="RightArrow" />
                                  </Col>
                                </Row>
                              </Col>
                              <Col
                                sm="5"
                                xs="5"
                                className="review-info receiver-review"
                              >
                                <div className="review-sub-block">
                                  <h3 className="review-title">
                                    Payout Method{" "}
                                  </h3>
                                  <div className="review-method">
                                    {this.props.receiverPaymentMethod
                                      .payoutMethod === "CASH_PICKUP" ? (
                                      <>
                                        <img
                                          src={CreditCardSvg}
                                          alt="CreditCardSvg"
                                        />
                                        Cash Pick{" "}
                                        <span
                                          onClick={() =>
                                            this.props.history.push(
                                              `${ROUTES.BENEFICIARY_DETAIL}`
                                            )
                                          }
                                        >
                                          ( Change )
                                        </span>{" "}
                                      </>
                                    ) : (
                                      <>
                                        <img
                                          src={BankoutSvg}
                                          alt="BankoutSvg"
                                        />
                                        Bank Deposit{" "}
                                        <span
                                          onClick={() =>
                                            this.props.history.push(
                                              `${ROUTES.BENEFICIARY_DETAIL}`
                                            )
                                          }
                                        >
                                          ( Change )
                                        </span>{" "}
                                      </>
                                    )}{" "}
                                  </div>
                                </div>
                                <div className="review-sub-block payment-info-show">
                                  {/* {this.props.receiverPaymentMethod
                                  .payoutMethod === "CASH_PICKUP" ? (
                                  <ReciverCashPickUpInfo
                                    cashPickUpInfo={
                                      this.props.receiverPaymentMethod
                                        .payoutDetail
                                    }
                                  />
                                ) : ( */}
                                  <ReciverBankInfo
                                    bankInfo={
                                      this.props.receiverPaymentMethod
                                        .payoutDetail
                                    }
                                  />
                                  {/* )} */}
                                </div>
                              </Col>
                            </Row>
                          </Card>
                        )}
                      </Col>
                    </Row>
                    <Row>
                      <Col className="pr-md-0">
                        <div
                          className="button-group mt-3"
                          style={{ maxWidth: "100%" }}
                        >
                          <button
                            onClick={() => {
                              this.sendMoney();
                            }}
                            className="btn btn-secondary"
                          >
                            {" "}
                            Send Money Now !
                          </button>
                          <button
                            className="btn btn-back-arrow float-right"
                            onClick={() => {
                              this.props.history.push(`${ROUTES.PAYMENT_INFO}`);
                            }}
                          >
                            <img src={BackArrow} alt="BackArrow" />
                            Back{" "}
                          </button>
                        </div>
                      </Col>
                    </Row>
                  </Col>
                  <Col
                    md={{ span: 8, offset: 2 }}
                    sm="12"
                    lg={{ span: 5, offset: 0 }}
                    className="sumary-bar"
                  >
                    <SendingMoneyDetail />
                  </Col>
                </Row>
              </Col>
            </Row>{" "}
          </Container>
        </section>

        <Footer />
        {!localStorage.acceptPolicy && <Disclaimer />}

        <AuthorizationModal
          isSubmitting={this.state.isSubmitting}
          show={this.state.showAuthorizationModal}
          close={this.closeModal}
          continue={this.continueSendMoney}
        />
        <CustomAlert
          show={this.state.showAlert}
          message={this.state.errorMessage}
          variant={this.state.variant}
          close={this.closeAlert}
        />
      </>
    );
  }
}
const mapActionToProps = {
  clearTransactionData,
  getTransactionAmount,
  getSenderTransactionLimit,
};

const mapStateToProps = (state) => ({
  user: state.auth.user,
  senderPaymentMethod: state.transaction.senderPaymentMethod,
  receiverPaymentMethod: state.transaction.receiverPaymentMethod,
  receiverInfo: state.transaction.receiverInfo,
  transactionDetail: state.transaction.transactionDetail,
  acceptPolicy: state.auth.accept_policy,
});

ReviewDetail.propTypes = {
  history: PropTypes.object,
  user: PropTypes.object,
  senderPaymentMethod: PropTypes.object,
  receiverPaymentMethod: PropTypes.object,
  receiverInfo: PropTypes.object,
  transactionDetail: PropTypes.object,
  clearTransactionData: PropTypes.func,
  getTransactionAmount: PropTypes.func,
  getSenderTransactionLimit: PropTypes.func,
};

export default connect(
  mapStateToProps,
  mapActionToProps
)(withRouter(ReviewDetail));
