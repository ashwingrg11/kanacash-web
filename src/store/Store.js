import { createStore, combineReducers, applyMiddleware, compose } from "redux";
// import { composeWithDevTools } from 'redux-devtools-extension';
import Thunk from "redux-thunk";
import LoginReducer from "./reducers/Login";
import ForgotPasswordReducer from "./reducers/ForgotPassword";
import LoaderReducer from "./reducers/Loader";
import ModalReducer from "./reducers/Modal";
import NetworkReducer from "./reducers/Network";
import AuthReducer from "./reducers/Auth";
import ErrorReducer from "./reducers/Error";
import MiscellaneousReducer from "./reducers/MiscellaneousReducer";
import senderDetailReducers from "./reducers/SenderDetailReducer";
import TransactionReducer from "./reducers/TransactionReducer";
import SignUpReducer from "./reducers/SignUpReducer";
import AdminReducers from "./reducers/adminReducers";

const middleware = [Thunk];
const initialState = {};

const composeEnhancers =
  typeof window === "object" && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
    ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
        trace: true,
      })
    : compose;

const rootReducer = combineReducers({
  login: LoginReducer,
  forgotPassword: ForgotPasswordReducer,
  loader: LoaderReducer,
  modal: ModalReducer,
  network: NetworkReducer,
  auth: AuthReducer,
  error: ErrorReducer,
  miscellaneous: MiscellaneousReducer,
  senderDetail: senderDetailReducers,
  transaction: TransactionReducer,
  signup: SignUpReducer,
  admin: AdminReducers,
});

const store = createStore(
  rootReducer,
  initialState,
  composeEnhancers(applyMiddleware(...middleware))
);
export default store;
