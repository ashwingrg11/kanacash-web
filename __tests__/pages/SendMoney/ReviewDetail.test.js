import React from "react";
import ReviewDetail from "../../../src/pages/SendMoney/ReviewDetail";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import { shallow } from "enzyme";
import store from "../../../src/store/Store";

import renderer from "react-test-renderer";
describe("<ReviewDetail />", () => {
  it("ReviewDetail renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <ReviewDetail />
          </Router>
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
