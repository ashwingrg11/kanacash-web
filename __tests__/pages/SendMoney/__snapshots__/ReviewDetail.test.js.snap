// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReviewDetail /> ReviewDetail renders correctly 1`] = `
Array [
  <header
    className="header-main"
  >
    <div
      className="container"
    >
      <nav
        className="navbar navbar-expand-lg navbar-light bg-light"
      >
        <div
          className="navbar-brand"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
          <img
            alt="logo"
            className="logo"
            src=""
          />
        </div>
        <button
          aria-controls="basic-navbar-nav"
          aria-label="Toggle navigation"
          className="navbar-toggler collapsed"
          onClick={[Function]}
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </button>
        <div
          aria-expanded={null}
          className="navbar-collapse collapse"
          id="basic-navbar-nav"
        >
          <div
            className="ml-md-auto navbar-nav"
            onKeyDown={[Function]}
          >
            <span
              className="btn btn-primary btn-theme btn-primary-border"
              onClick={[Function]}
            >
              Sign in
            </span>
            <button
              className="btn btn-primary btn-theme btn-primary-theme"
              onClick={[Function]}
            >
              Sign up
            </button>
          </div>
        </div>
      </nav>
    </div>
  </header>,
  <div
    className="shadow-line"
  />,
  <section
    className="theme-inner"
  >
    <div
      className="container"
    >
      <div
        className="transaction-steps row"
      >
        <div
          className="col"
        >
          <div
            className="d-flex justify-content-left align-items-center"
          >
            <div
              className="green-block payment-step"
            >
               
              <img
                alt="paymentDetailGreen"
                src=""
              />
              <span
                className="steps-title"
              >
                Payment Details
              </span>
               
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="d-flex justify-content-left align-items-center"
          >
            <div
              className="gray-block sender-step"
            >
               
              <img
                alt="SenderDetailGrey"
                src=""
              />
              <span
                className="steps-title"
              >
                Sender Details
              </span>
            </div>
             
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="d-flex justify-content-left align-items-center"
          >
             
            <div
              className="gray-block beneficiary-step"
            >
              <img
                alt="BeneficiaryDetailGrey"
                src=""
              />
              <span
                className="steps-title"
              >
                Beneficiary Details
              </span>
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="d-flex justify-content-left align-items-center"
          >
             
            <div
              className="gray-block pay-info-step"
            >
              <img
                alt="paymentInfoGrey"
                src=""
              />
              <span
                className="steps-title"
              >
                Payment Information
              </span>
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="d-flex justify-content-left align-items-center"
          >
            <div
              className="gray-block review-step last-step"
            >
              <img
                alt="ReviewDetailGrey"
                src=""
              />
              <span
                className="steps-title"
              >
                Review Details
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="steps-content-block col"
        >
          <h2
            className="sub-title text-md-center text-lg-left"
          >
             
            Here is your transaction summary
          </h2>
          <p
            className="text-md-center text-lg-left"
            style={
              Object {
                "fontSize": "18px",
              }
            }
          >
            Please review to make sure everything is correct before proceeding.
          </p>
          <div
            className="row"
          >
            <div
              className="pr-lg-0 mb-md-5 mb-lg-0 col-lg-7 col-md-8 col-sm-12 offset-lg-0 offset-md-2"
            >
              <div
                className="row"
              >
                <div
                  className="pr-xl-0 col"
                >
                  <div
                    className="card"
                    style={
                      Object {
                        "padding": "20px",
                      }
                    }
                  >
                    <div
                      className="row"
                    >
                      <div
                        className="review-info sender-review col-sm-5 col-5"
                      >
                        <h3
                          className="review-title green-title"
                        >
                           
                          Sender Information
                        </h3>
                        <div
                          className="user-detail-review transaction-pty"
                        >
                          <h4>
                             
                          </h4>
                          <p />
                        </div>
                      </div>
                      <div
                        className="arrow text-center col-sm-2 col-2"
                      >
                         
                        <div
                          className="row"
                          style={
                            Object {
                              "marginTop": "30px",
                            }
                          }
                        >
                          <div
                            className="col"
                          >
                            <img
                              alt="RightArrow"
                              src=""
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        className="review-info receiver-review col-sm-5 col-5"
                      >
                        <h3
                          className="review-title green-title"
                        >
                           
                          Recipient Information
                        </h3>
                        <div
                          className="user-detail-review transaction-pty"
                        >
                          <h4>
                             
                            <span
                              className="change"
                              onClick={[Function]}
                            >
                              (Change)
                            </span>
                          </h4>
                          <p>
                            ,
                             
                          </p>
                        </div>
                      </div>
                    </div>
                    <div
                      className="row"
                    >
                      <div
                        className="review-info sender-review col-sm-5 col-5"
                      >
                        <div
                          className="review-sub-block"
                        >
                          <h3
                            className="review-title"
                          >
                            Payment Method
                             
                          </h3>
                          <div
                            className="review-method"
                          >
                            <img
                              alt="BankLogoSvg"
                              src=""
                            />
                            Bank
                             
                            <span
                              onClick={[Function]}
                            >
                              ( Change )
                            </span>
                             
                             
                          </div>
                        </div>
                        <div
                          className="review-sub-block payment-info-show"
                        >
                          <div
                            className="user-detail-review bankinfo-sender"
                          >
                            <p>
                               
                               
                            </p>
                            <p>
                               
                              <span>
                                Account Holder
                              </span>
                               
                              <span>
                                 
                              </span>
                            </p>
                            <p>
                              <span>
                                Account Type
                              </span>
                               
                            </p>
                          </div>
                        </div>
                      </div>
                      <div
                        className="arrow text-center col-sm-2 col-2"
                      >
                         
                        <div
                          className="row"
                          style={
                            Object {
                              "marginTop": "60px",
                            }
                          }
                        >
                          <div
                            className="col"
                          >
                            <img
                              alt="RightArrow"
                              src=""
                            />
                          </div>
                        </div>
                      </div>
                      <div
                        className="review-info receiver-review col-sm-5 col-5"
                      >
                        <div
                          className="review-sub-block"
                        >
                          <h3
                            className="review-title"
                          >
                            Payout Method
                             
                          </h3>
                          <div
                            className="review-method"
                          >
                            <img
                              alt="BankoutSvg"
                              src=""
                            />
                            Bank Deposit
                             
                            <span
                              onClick={[Function]}
                            >
                              ( Change )
                            </span>
                             
                             
                          </div>
                        </div>
                        <div
                          className="review-sub-block payment-info-show"
                        >
                          <div
                            className="user-detail-review bankinfo-sender"
                          >
                            <p>
                               
                               
                            </p>
                            <p>
                              <span>
                                Account Number
                              </span>
                            </p>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="row"
              >
                <div
                  className="pr-md-0 col"
                >
                  <div
                    className="button-group mt-3"
                    style={
                      Object {
                        "maxWidth": "100%",
                      }
                    }
                  >
                    <button
                      className="btn btn-secondary"
                      onClick={[Function]}
                    >
                       
                      Send Money Now !
                    </button>
                    <button
                      className="btn btn-back-arrow float-right"
                      onClick={[Function]}
                    >
                      <img
                        alt="BackArrow"
                        src=""
                      />
                      Back
                       
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="sumary-bar col-lg-5 col-md-8 col-sm-12 offset-lg-0 offset-md-2"
            >
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <div
                    className="summary-card card"
                  >
                     
                    <h2
                      className="sub-title"
                    >
                      Summary
                    </h2>
                     
                    <p
                      className="small"
                    >
                       Sending money to
                    </p>
                     
                    <div
                      className="send-country"
                    >
                       
                      <span
                        className="country-flag"
                      >
                        <img
                          alt="flag"
                          src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.2.1/flags/4x3/undefined.svg"
                        />
                      </span>
                       
                      <span
                        className="sub-title"
                      />
                       
                    </div>
                    <div>
                      <hr
                        style={
                          Object {
                            "borderColor": "#212529",
                          }
                        }
                      />
                    </div>
                    <div
                      className="row"
                    >
                      <div
                        className="col"
                      >
                        <div
                          className="mt-3 row"
                        >
                          <div
                            className="col"
                          >
                            Send Amount
                             
                            <span
                              className="btn-link"
                              onClick={[Function]}
                              style={
                                Object {
                                  "color": "#1077BE",
                                  "cursor": "pointer",
                                }
                              }
                            >
                              edit
                            </span>
                          </div>
                          <div
                            className="d-flex justify-content-end col"
                            style={
                              Object {
                                "fontWeight": "bold",
                              }
                            }
                          >
                            USD 
                          </div>
                        </div>
                        <div
                          className="mt-3 row"
                        >
                          <div
                            className="col"
                          >
                            Transaction Fee
                          </div>
                          <div
                            className="d-flex justify-content-end col"
                            style={
                              Object {
                                "fontWeight": "bold",
                              }
                            }
                          >
                            USD 
                          </div>
                        </div>
                        <div
                          className="mt-3 row"
                        >
                          <div
                            className="col"
                          >
                            Total Amount
                          </div>
                          <div
                            className="d-flex justify-content-end col"
                            style={
                              Object {
                                "color": "#1077BE",
                                "fontWeight": "bold",
                              }
                            }
                          >
                            USD
                             
                            NaN
                          </div>
                        </div>
                        <div
                          className="d-flex align-items-center mt-3 xchange-rate row"
                          style={
                            Object {
                              "backgroundColor": "#d1ecf1",
                              "height": "40px",
                            }
                          }
                        >
                          <div
                            className="col"
                          >
                            Exchange Rate
                          </div>
                          <div
                            className="d-flex justify-content-end col"
                            style={
                              Object {
                                "fontWeight": "bold",
                              }
                            }
                          >
                            USD 1 =
                             
                             
                          </div>
                        </div>
                        <div
                          className="mt-3 row"
                        >
                          <div
                            className="col"
                          >
                            Receive Amount
                          </div>
                          <div
                            className="d-flex justify-content-end col"
                            style={
                              Object {
                                "fontWeight": "bold",
                              }
                            }
                          >
                             
                          </div>
                        </div>
                        <div
                          className="mt-3 row"
                        >
                          <div
                            className="col"
                          >
                            Payment Type
                          </div>
                          <div
                            className="d-flex justify-content-end col"
                            style={
                              Object {
                                "fontWeight": "bold",
                              }
                            }
                          >
                            TBD
                          </div>
                        </div>
                        <div
                          className="mt-3 row"
                        >
                          <div
                            className="col"
                          >
                            Receive Type
                          </div>
                          <div
                            className="d-flex justify-content-end col"
                            style={
                              Object {
                                "fontWeight": "bold",
                              }
                            }
                          >
                            TBD
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
    </div>
  </section>,
  <footer
    className="footer-section text-center"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-12"
        >
          <div
            className="foot-nav"
          >
            <a
              aria-current="page"
              className="link active"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
            <a
              aria-current={null}
              className="link"
              href="/about-us"
              onClick={[Function]}
            >
              About Us
            </a>
            <a
              aria-current={null}
              className="link"
              href="/terms-of-service"
              onClick={[Function]}
            >
              Terms
            </a>
            <a
              aria-current={null}
              className="link"
              href="/privacy-policy"
              onClick={[Function]}
            >
              Privacy Policy
            </a>
            <a
              aria-current={null}
              className="link"
              href="/copyright"
              onClick={[Function]}
            >
              Copyright
            </a>
            <a
              aria-current={null}
              className="link"
              href="/contact-us"
              onClick={[Function]}
            >
              Contact
            </a>
          </div>
        </div>
        <div
          className="copyright col-sm-12"
        >
          <p>
            © Golden Money Transfer Inc. 2020. All Rights Reserved.
             
            <b>
              Licensed by
            </b>
             Department of Financial Institutions, Nevada
          </p>
        </div>
      </div>
    </div>
  </footer>,
  <div
    className="disclaimer text-center"
  >
    <div
      className="container"
    >
      <div
        className="policy"
      >
         
        This website uses cookies to ensure you get the best experience on our website, as described in our
         
        <span
          className="btn-link"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
           
            privacy policy.
        </span>
      </div>
      <div
        className="policy1"
      >
        This website uses cookies,
        <span
          className="btn-link"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
           
            learn more.
        </span>
      </div>
      <div
        className="buttonDiv"
      >
        <button
          className="btn-outline-warning"
          onClick={[Function]}
          style={
            Object {
              "border": "none",
              "borderRadius": "50px",
              "color": "black",
              "padding": "5px 15px",
            }
          }
        >
          Accept
        </button>
      </div>
    </div>
  </div>,
]
`;
